src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts

```diff
@@ -842,7 +842,6 @@ export class TerminalTaskSystem extends Disposable implements ITaskSystem {
 					if (eventCounter === 0) {
 						if ((watchingProblemMatcher.numberOfMatches > 0) && watchingProblemMatcher.maxMarkerSeverity &&
 							(watchingProblemMatcher.maxMarkerSeverity >= MarkerSeverity.Error)) {
-							this._fireTaskEvent(TaskEvent.general(TaskEventKind.ProblemMatcherFoundErrors, task, terminal?.instanceId));
```

Comment:
thanks for investigating - we don't want to remove this - as it's needed for the API. Also, there should be no reason this causes an issue with debugging. if it does fix it , can you pls debug to see why?